__tests__/app.test.js

```diff
@@ -0,0 +1,32 @@
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+const request = require("supertest");
+const app = require("../db/app");
+
+beforeAll(() => seed(data));
+afterAll(() => db.end());
+
+describe("Get/api/topics", () => {
+  test("response with the status of 200 and all the available topics ", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        body.topics.map((topic) => {
+          expect(topic).toMatchObject({
+            description: expect.any(String),
+            slug: expect.any(String),
+          });
+        });
```

Comment:
If topics is an empty array map (as well as any other array method) will actually be skipped and wont run so we also need to include another expect statement outside of this map to ensure that the array has some items within it. I would recommending looking into the .toBeGreaterThan matcher for this.